TYPO3.Flow/Classes/TYPO3/Flow/Validation/Validator/AggregateBoundaryValidator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace TYPO3\Flow\Validation\Validator;
+
+/*
+ * This file is part of the TYPO3.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+use Doctrine\ORM\Proxy\Proxy;
+
+/**
+ * A validator which will only validate up to the Aggregate boundary, or if an Aggregate relation
+ * is specifically annotated to be cascade validated.
```

Comment:
This needs a correction, as there currently isn't a simple cascade validation annotation.